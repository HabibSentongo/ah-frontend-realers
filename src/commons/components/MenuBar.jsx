import React from 'react';
import { Nav } from 'react-bootstrap';
import '../CSS/Header.scss';
import { Link } from 'react-router-dom';

export default function MenuBar() {
  return (
    <>
      <Nav className="justify-content-center menu-section" activeKey="/">
        <Link to="/">
          <Nav.Item>
            HOME
          </Nav.Item>
        </Link>
        <Link to="/latest">
          <Nav.Item>
            LATEST
          </Nav.Item>
        </Link>
        <Link to="/top">
          <Nav.Item>
            TOP
          </Nav.Item>
        </Link>
        <Link to="/about">
          <Nav.Item>
            ABOUT
          </Nav.Item>
        </Link>
      </Nav>
    </>
  );
}
