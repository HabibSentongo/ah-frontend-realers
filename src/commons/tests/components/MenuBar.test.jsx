/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MenuBar from '../../components/MenuBar';

// test
describe('MenuBar Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<MenuBar />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
