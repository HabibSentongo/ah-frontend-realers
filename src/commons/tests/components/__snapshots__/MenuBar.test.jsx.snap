// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuBar Component should match the snapshot 1`] = `
<Fragment>
  <Nav
    activeKey="/"
    className="justify-content-center menu-section"
    fill={false}
    justify={false}
  >
    <Link
      to="/"
    >
      <NavItem>
        HOME
      </NavItem>
    </Link>
    <Link
      to="/latest"
    >
      <NavItem>
        LATEST
      </NavItem>
    </Link>
    <Link
      to="/top"
    >
      <NavItem>
        TOP
      </NavItem>
    </Link>
    <Link
      to="/about"
    >
      <NavItem>
        ABOUT
      </NavItem>
    </Link>
  </Nav>
</Fragment>
`;
